buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -30,8 +34,12 @@ object Versions {
 
     const val espresso_core: String = "3.2.0"
 
+    const val fragment_ktx: String = "1.2.5"
+
     const val lint_gradle: String = "27.2.0-alpha02"
 
+    const val viewbinding: String = "4.2.0-alpha02"
```

Comment:
might as well use Compose too since you are using so many non final api libraries